test.js

```diff
@@ -11,15 +11,6 @@ function deserializeNonError(t, value) {
 	t.is(deserialized.message, JSON.stringify(value));
 }
 
-// TODO: Replace with plain `new Error('outer', {cause: new Error('inner')})` when targeting Node 16.9+
```

Comment:
There's another TODO, but the mentioned API is still marked as "experimental"

https://nodejs.org/api/stream.html#streamisreadablestream

https://github.com/sindresorhus/serialize-error/blob/a2df3aeea73e39482eeb7d2fe445ac7120231593/index.js#L101-L105